### PR TITLE
Added Auto Signal based on feedback

### PR DESCRIPTION
**Pull request by UmangSharmaMobis**
added Auto signal based on feedback received https://github.com/COVESA/vehicle_signal_specification/issues/661

### CODE REVIEW

File: spec/include/PowerOptimize.vspec [hunk 1]
```diff
@@ -107,3 +107,12 @@ PowerOptimizeLevel:
...
                Level 10 indicates most aggressive power optimization mode, only essential functionality enabled.
+
+AutoPowerOptimize:
+  datatype: boolean
+  type: actuator
+  description: Auto Power Optimization Flag
+               When set to 'true', the system enables automatic power optimization,
+               dynamically adjusting the power optimization level based on runtime conditions
+               or features managed by the OEM.
+               When set to 'false', manual control of the power optimization level is allowed.
```
Comment (erikbosch) [inline spec/include/PowerOptimize.vspec:118]: Newline required to satisfy linter check

**Comment by erikbosch** [comment]
Meeting notes:

- Umang to fix CI errors
- Please review, merge candidate for next meeting

**Comment by UmangSharmaMobis** [comment]
will FIx CI errors in New Pull Request

**Comment by UmangSharmaMobis** [comment]
will FIx CI errors in New Pull Request

**Pull request closed without merging**